static/src/javascripts-legacy/projects/common/modules/experiments/ab.js

```diff
@@ -105,14 +104,15 @@ define([
     }
 
     function getExpiredTests() {
-        var now = new Date();
+        var startOfToday = new Date().setHours(0,0,0,0);
         return TESTS.filter(function (test) {
-            return (now - new Date(test.expiry)) > 0;
+            return (startOfToday > new Date(test.expiry));
         });
     }
 
     function testCanBeRun(test) {
-        var expired = (new Date() - new Date(test.expiry)) > 0,
+        var startOfToday = new Date().setHours(0,0,0,0);
+        var expired = (startOfToday > new Date(test.expiry)),
```

Comment:
i would create a separate `isExpired` function and use it here and in the filter